__tests__/app.test.js

```diff
@@ -33,4 +33,41 @@ describe("/api/topics", () => {
                 });
         });
     });
-});
\ No newline at end of file
+});
+describe('/api/articles/:article_id', () => {
+    describe('GET', () => {
+        test("200: returns specific article", () => {
+            return request(app)
+                .get(`/api/articles/1`)
+                .expect(200)
+                .then(({ body }) => {
+                    expect(body.article).toBeInstanceOf(Object);
+                    expect(body.article[0]).toMatchObject({
```

Comment:
You'll just have to tweak this when you adjust it to send back the object, instead of the array with thee one object